.github/workflows/test-notify-slack-workflow.yml

```
@@ -0,0 +1,25 @@
+---
+
+name: 'Run notify-slack workflow'
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/notify-slack.yml
+      - .github/actions/test-notify-slack/*
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  run-notify-slack-workflow:
+    uses: ./.github/workflows/notify-slack.yml@feature/slack-notify
+    with:
+      application: github-actions
+      environment: staging
+      status: ${{ inputs.status}}
```

This workflow has no inputs.

```suggestion
      status: test
```